client/components/videos-ui/index.jsx

```diff
@@ -117,11 +117,12 @@ const VideosUi = ( { headerBar, footerBar } ) => {
 					</div>
 				</div>
 			</div>
-			<div className="videos-ui__body">
+			<div className={ `videos-ui__body ${ course ? '' : 'is-loading' }` }>
```

Comment:
Can you use `classNames` for this? It will allow conditional classes like this, but won't leave stray whitespace in the markup when `is-loading` isn't there.